.eslintignore

```diff
@@ -3,4 +3,4 @@ build
 package.json
 webpack.config.js
 cypress.config.js
-src/constants.js
+src/demos.js
```

Comment:
could this be removed and use `// eslint-disable-next-line` for the specific lines in `demos.js` instead? In case we forget that the whole file is ignored in the future

Reply:
Removing all together